volume/properties/common.properties

```diff
@@ -13,6 +13,11 @@ org.openelisglobal.fhirstore.username=admin
 org.openelisglobal.fhirstore.password=adminADMIN!
 org.openelisglobal.fhir.subscriber.allowHTTP=true
 
+#open client registry authentication
+org.openelisglobal.crserver.uri=http://localhost:5001/CR/fhir
```

Comment:
@reagan-meant this configuration seems to return this error when i boot up the docker compose with opencr, openhim, etc and this docker compose with openelis: Happens when i attempt to created a patient, the patient is created but server logs are as below indicating a connection issue!

```
ERROR -- Class: SimpleAsyncTaskExecutor-15, Method: transformPersistPatient, Error: HAPI-1357: Failed to retrieve the server metadata statement during client initialization. URL used was http://localhost:5001/CR/fhir/metadata
openelisglobal-webapp  | 2024-07-04T11:46:03.261+0300 -- ERROR -- Class: ca.uhn.fhir.rest.client.impl.RestfulClientFactory, Method: validateServerBase, Line: 332, Message: HAPI-1357: Failed to retrieve the server metadata statement during client initialization. URL used was http://localhost:5001/CR/fhir/metadata
openelisglobal-webapp  | Class: ca.uhn.fhir.rest.client.impl.BaseClient, Method: invokeClient, Line: 418, Sub-Message: HAPI-1361: Failed to parse response from server when performing GET to URL http://localhost:5001/CR/fhir/metadata - org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused (Connection refused)
openelisglobal-webapp  | Class: org.apache.http.impl.conn.DefaultHttpClientConnectionOperator, Method: connect, Line: 156, Sub-Message: Connect to localhost:5001 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused (Connection refused)
```

Reply:
@mherman22 could you share all the server logs on paste bin.

Reply:
https://pastebin.ubuntu.com/p/zcnk62rSqg/